lib/node_modules/@stdlib/string/base/truncate-middle/README.md

```diff
@@ -0,0 +1,92 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2023 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# truncateMiddle
+
+> Truncate the middle UTF-16 code units of a string to a specified length.
```

Comment:
```suggestion
> Truncate the middle UTF-16 code units of a string to return a string having a specified length.
```

@steff456 Sorry to be pedantic. Would you mind changing the descriptions based on the above so that we are a bit more clear? Currently, the description makes it seem as though the function only reduces the middle UTF-16 units to a specified length, not the overall string.